Library/Homebrew/diagnostic.rb

```diff
@@ -835,16 +834,29 @@ def check_deleted_formula
         kegs = Keg.all
 
         deleted_formulae = kegs.map do |keg|
-          next if Formulary.tap_paths(keg.name).any?
+          tap = Tab.for_keg(keg).tap
+
+          loadable = [
+            Formulary::FromAPILoader,
+            Formulary::FromNameLoader,
+          ].any? do |loader_class|
```

Comment:
Does `FromDefaultNameLoader` need to be added here?

Reply:
It should work without it because of the `warn: false`, but doesn't hurt to add it here.